addons/website_sale/controllers/variant.py

```diff
@@ -1,33 +1,47 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import http
-from odoo.http import request
+# Part of Odoo. See LICENSE file for full copyright and licensing details
 
-from odoo.addons.sale.controllers.variant import VariantController
+import json
 
+from odoo.http import request, route, Controller
 
-class WebsiteSaleVariantController(VariantController):
-    @http.route(['/sale/get_combination_info_website'], type='json', auth="public", methods=['POST'], website=True)
+
+class WebsiteSaleVariantController(Controller):
+
+    @route('/sale/get_combination_info_website', type='json', auth="public", methods=['POST'], website=True)
     def get_combination_info_website(self, product_template_id, product_id, combination, add_qty, **kw):
-        """Special route to use website logic in get_combination_info override.
-        This route is called in JS by appending _website to the base route.
-        """
-        kw.pop('pricelist_id')
-        combination = self.get_combination_info(product_template_id, product_id, combination, add_qty, request.website.get_current_pricelist(), **kw)
+        """Old product configurator logic, only used by frontend configurator rn, will be dropped soon"""
```

Comment:
"is deprecated"?

Anyway, this doesn't look like a docstring we want ^^'

Reply:
She doesn't like my beautiful docstring :sob:

Reply:
Sorry not sorry =P